app/src/main/java/ch/sdp/vibester/fragment/SearchUserFragment.kt

```diff
@@ -47,34 +50,25 @@ class SearchUserActivity : AppCompatActivity(), OnItemClickListener {
 
     private var uidList: ArrayList<String> = ArrayList()
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        requestWindowFeature(Window.FEATURE_NO_TITLE)
-        super.onCreate(savedInstanceState)
-        supportActionBar?.hide()
-        setContentView(R.layout.activity_search_user)
-
-        recyclerView = findViewById(R.id.searchList)
+        recyclerView = view.findViewById(R.id.searchList)
         recyclerView!!.setHasFixedSize(true)
-        recyclerView!!.layoutManager = LinearLayoutManager(this)
+        recyclerView!!.layoutManager = LinearLayoutManager(requireContext())
 
         userProfileAdapter = UserProfileAdapter(this.users, authenticator, usersRepo, imageGetter, this)
 
         recyclerView!!.adapter = userProfileAdapter
 
-        searchEditText = findViewById(R.id.searchUserET)
+        searchEditText = view.findViewById(R.id.searchUserET)
```

Comment:
Is the "view" necessary here?

Reply:
yes, cause it is fragment, and not an activity

Reply:
fragments have different way to manage view, context and activity